app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -126,6 +123,13 @@
     private ImageView arcPointer;
     private LinearLayout infoLayout;
 
+    private LinearLayout touchView;
+    private WindowManager.LayoutParams touchViewParams;
+    private Handler screenScanHandler;
+    private Runnable screenScanRunnable;
```

Comment:
Why final? I want to release them when the service isn't running or not allocate them when we are using battery saver mode.